lib/cli/run-helpers.js

```diff
@@ -180,11 +208,14 @@ exports.runMocha = async (mocha, options) => {
     spec
   };
 
+  let run;
   if (watch) {
-    watchRun(mocha, {watchFiles, watchIgnore}, fileCollectParams);
+    run = parallel ? watchParallelRun : watchRun;
   } else {
-    await singleRun(mocha, {exit}, fileCollectParams);
+    run = parallel ? parallelRun : singleRun;
   }
+
```

Comment:
my assumption is that people tend not to love nested ternaries. but if someone wants it, I will abide